packages/sampler/stories/default/VirtualList.js

```diff
@@ -17,17 +17,26 @@ const
 		'&quot;noAnimation&quot;': 'noAnimation'
 	},
 	prop = {
+		direction: {horizontal: 'horizontal', vertical: 'vertical'},
```

Comment:
This can be simplified to just an array: `direction: ['horizontal', 'vertical']`

Reply:
Oops!